### PR TITLE
Sprint i18n

### PR DESCRIPTION
**Pull request by VIVO-UQAM**
Ajustement Java 1.8 + managing UTF-8 from pom files

### CODE REVIEW

**Comment by awoods** [comment]
It appears that this pull-request contains several conflicts:
https://github.com/vivo-project/Vitro-languages/pull/11/files

Can you close this PR and resubmit a clean version?

**Review by VIVO-UQAM — COMMENTED** [review]
File Unscrambling

File: core/webapp/pom.xml [hunk 1]
```diff
@@ -15,7 +16,13 @@
     </parent>
 
     <name>Vitro Languages for Web App CORE</name>
-
+    
+    <properties>
```
Comment (awoods) [inline core/webapp/pom.xml]: If we update the top-level pom.xml with these additional properties, then all `<modules>` will inherit them, no?
https://github.com/vivo-project/Vitro-languages/blob/sprint-i18n/pom.xml#L55-L63

**Review by awoods — CHANGES_REQUESTED** [review]

**Comment by awoods** [comment]
There still appear to be merge conflicts. I would suggest updating your `UQAM-VIVO/Vitro-languages-1` branch to be up to date with `sprint-i18n`, then add your changes on top of that.

File: core/webapp/pom.xml (hunk 1 above)
Comment (VIVO-UQAM) [reply core/webapp/pom.xml]: I'am not shure. I'll run some tests and I'll come back with the solution.

**Comment by awoods** [comment]
It might be easiest to close this PR, and create a new PR with only the following changes:
https://github.com/vivo-project/Vitro-languages/pull/11/files#diff-1efe745723b9e36904a4f3615f53d61aR7

https://github.com/vivo-project/Vitro-languages/pull/11/files#diff-600376dffeb79835ede4a0b285078036R63-R64

**Pull request closed without merging**